### PR TITLE
Update docs for the timestamp function (introduced in T0678-MG)

### PR DESCRIPTION
**Pull request by kostasrim**

### CODE REVIEW

**Comment by antonio2368** [comment]
Update the CHANGELOG also.

**Comment by kostasrim** [comment]
@g-despot  Shall I also update the `/versioned` docs?

**Comment by antaljanosbenjamin** [comment]
> @g-despot Shall I also update the `/versioned` docs?

I am not Ivan, but I know the answer! You shouldn't. The `/versioned` docs are for previous versions. As the `timestamp` was returning milliseconds before 2.0, the versioned docs should be untouched.

File: docs/changelog.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.0.0 - TBA
+
+:::warning
+### Breaking Changes
+* Changed the `Timestamp()` function to return `microseconds` instead of `milliseconds`.
```
Comment (jbajic) [inline docs/changelog.md:11]:
```suggestion
* Changed the `timestamp()` function to return `microseconds` instead of `milliseconds`.
```

**Pull request merged**